src/core/net/mdns.cpp

```diff
@@ -2948,6 +2985,18 @@ void Core::TxMessage::AppendServicesDnssdName(Section aSection)
     return;
 }
 
+void Core::TxMessage::AddQuestionFrom(OwnedPtr<Message> &aMessage)
+{
+    uint16_t offset = sizeof(Header);
+
+    IgnoreError(Name(*aMessage, sizeof(Header)).AppendTo(*mMsgPtr));
+    IgnoreError(Name::ParseName(*aMessage, offset));
+    // RFC6762, section 6.7:
+    // Legacy Unicast Response must repeat the question
```

Comment:
Remove this comment here?  
```suggestion
```

Can consider moving this to where we call this method from `Core::RxMessage::SendUnicastResponse()`?

Reply:
Agree, will move this in ```Core::RxMessage::SendUnicastResponse()``` under ```if (mIsLegacyUnicast)```.